src/H5Cpkg.h

```diff
@@ -3409,227 +2721,196 @@ typedef struct H5C_tag_info_t {
+ * size_decreases: Array to record the number of times an entry
  *        with type id equal to the array index has decreased in
  *        size in the current epoch.
  *
- * entry_flush_size_changes:  Array of int64 of length
- *         H5C__MAX_NUM_TYPE_IDS + 1.  The cells are used to record
- *         the number of times an entry with type id equal to the
- *         array index has changed size while in its pre-serialize
- *        callback.
+ * entry_flush_size_changes: Array to record the number of times an entry
+ *        with type id equal to the array index has changed size while in
+ *        its pre-serialize callback.
  *
- * cache_flush_size_changes:  Array of int64 of length
- *         H5C__MAX_NUM_TYPE_IDS + 1.  The cells are used to record
- *         the number of times an entry with type id equal to the
- *         array index has changed size during a cache flush
+ * cache_flush_size_changes: Array to record the number of times an entry
+ *        with type id equal to the array index has changed size during a
+ *        cache flush
  *
  * total_ht_insertions: Number of times entries have been inserted into the
  *        hash table in the current epoch.
  *
  * total_ht_deletions: Number of times entries have been deleted from the
- *              hash table in the current epoch.
- *
- * successful_ht_searches: int64 containing the total number of successful
- *        searches of the hash table in the current epoch.
+ *        hash table in the current epoch.
  *
- * total_successful_ht_search_depth: int64 containing the total number of
- *        entries other than the targets examined in successful
- *        searches of the hash table in the current epoch.
+ * successful_ht_searches: The total number of successful searches of the
+ *        hash table in the current epoch.
  *
- * failed_ht_searches: int64 containing the total number of unsuccessful
- *              searches of the hash table in the current epoch.
+ * total_successful_ht_search_depth: The total number of entries other than
+ *        the targets examined in successful searches of the hash table in
+ *        the current epoch.
  *
- * total_failed_ht_search_depth: int64 containing the total number of
- *              entries examined in unsuccessful searches of the hash
+ * failed_ht_searches: The total number of unsuccessful searches of the hash
  *        table in the current epoch.
  *
- * max_index_len:  Largest value attained by the index_len field in the
- *              current epoch.
+ * total_failed_ht_search_depth: The total number of entries examined in
+ *        unsuccessful searches of the hash table in the current epoch.
+ *
+ * max_index_len: Largest value attained by the index_len field in the
+ *        current epoch.
  *
- * max_index_size:  Largest value attained by the index_size field in the
- *              current epoch.
+ * max_index_size: Largest value attained by the index_size field in the
+ *        current epoch.
```

Comment:
> It'd be nice if all this could be made to line up

Agree, but I don't want to hold this PR on that.  I'll try to polish them over the next few PRs